src/services/federalState/countyModel.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const countySchema = new Schema(
+	{
+		countyId: { type: Number },
+		county: { type: String },
+		antaresKey: { type: String },
+	},
+	{
+		timestamps: true,
+	}
+);
+module.exports = {
+	CountyModel: mongoose.model('county', countySchema),
```

Comment:
Please do not export the model and if the schema is only embedded on federalState at it to it.